packages/pn-personafisica-webapp/src/components/Contacts/SpecialContacts.tsx

```diff
@@ -221,6 +247,27 @@ const SpecialContacts: React.FC<{ addressType: AddressType; channelType?: Channe
   };
 
   const handleDelete = (value: string, channelType: ChannelType, sender: Sender) => {
+    PFEventStrategyFactory.triggerEvent(
+      channelType === ChannelType.SERCQ_SEND
+        ? PFEventsType.SEND_REMOVE_SERCQ_SEND_START
+        : PFEventsType.SEND_REMOVE_DIGITAL_DOMICILE_PEC_START,
+      {
+        event_type: EventAction.ACTION,
+        addresses,
+        other_contact: true,
+      }
+    );
+
+    PFEventStrategyFactory.triggerEvent(
```

Comment:
can be moved under the setModalOpen call?

Reply:
I meant to move this call under the row with the code setModalOpen(ModalType.DELETE);